package.json

```diff
@@ -16,32 +16,30 @@
   "dependencies": {
     "easy-bem-naming": "^1.0.2",
     "inherit": "2.2.6",
-    "prop-types": "15.5.8"
+    "preact-render-spy": "^1.1.0"
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-core": "^6.24.1",
     "babel-eslint": "^7.2.3",
     "babel-jest": "^19.0.0",
     "babel-plugin-bem-import": "^1.4.1",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-plugin-transform-object-assign": "^6.22.0",
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "enzyme": "^2.8.2",
     "eslint": "^3.19.0",
     "eslint-plugin-jest": "^19.0.1",
     "eslint-plugin-react": "^6.10.3",
     "husky": "^0.13.3",
     "jest": "^19.0.2",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
-    "react-addons-test-utils": "^15.5.1",
-    "react-test-renderer": "^15.5.4"
+    "preact": "^8.2.5",
+    "proptypes": "^1.1.0"
   },
   "peerDependencies": {
-    "react": "^15.0.0-0 || ^16.0.0-0",
-    "react-dom": "^15.0.0-0 || ^16.0.0-0"
+    "preact": "^8.2.5",
+    "proptypes": "^1.1.0"
```

Comment:
In the proptypes readme I see:

> This is literally copied from the React 0.14 codebase

wtf? preact doesn't work properly with prop-types module?

Reply:
nope, see docs: https://preactjs.com/guide/differences-to-react#whats-missing

Reply:
Holy hell!